### PR TITLE
CCloudify cogrouping streams KT

### PR DESCRIPTION
**Pull request by awalther28**
### Description

CCloudify cogrouping streams KT. Added short answer to the Confluent and kstreams KTs. Validated cloud steps locally. 

Given difficulties with conversion, this PR suggests removing some content and replacing it with a data generator in the kstreams application. 

Testing passed locally and in semaphore. Failing tests in the latest commit/semaphore run are unrelated. 

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/convert-cogrouping-streams/cogrouping-streams/confluent.html

### New tutorial checklist

- [x] Add a `Short Answer`
- [ ] Implement good test cases
- [ ] Validate hyperlinks
- [ ] Spell check (e.g. using `aspell`)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -117,7 +117,7 @@ <h4>All the tutorials can be run locally or with <a href="https://www.confluent.
             <li><a href="create-stateful-aggregation-sum/ksql.html">Sum a stream of events</a></li>
             <li><a href="create-stateful-aggregation-minmax/ksql.html">Find the min/max</a></li>
             <li><a href="aggregating-average/kstreams.html">Calculate a running average</a></li>
-            <li><a href="cogrouping-streams/kstreams.html">Cogrouping aggregates</a></li>
+            <li><a href="cogrouping-streams/confluent.html">Cogrouping aggregates</a></li>
```
Comment (ybyzek) [inline index.html:120]: Great!

File: _includes/tutorials/cogrouping-streams/confluent/markup/answer/short-answer.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Use a combination of `cogroup` and and `aggregate` functions as seen below.
```
Comment (ybyzek) [inline _includes/tutorials/cogrouping-streams/confluent/markup/answer/short-answer.adoc:1]:
```suggestion
Use a combination of `cogroup` and and `aggregate` methods as shown below.
```

**Comment by bbejeck** [comment]
Test for this PR passed https://confluentinc.semaphoreci.com/jobs/21a6cc09-5a23-4b83-9548-4377af4691f1

**Review by bbejeck — COMMENTED** [review]
Thanks for the PR @awalther28 looks good overall.  

I encountered one issue that I think has been there from before.  If users don't stop the Kafka Streams app from running and they move on to the test section, the unit test fails because it can't get a lock on the state directory.  If we add this line
```
state.dir=cogrouping-test-state
```
then it should be ok regardless if the steams app is shutdown or not.

**Comment by awalther28** [comment]
@bbejeck I have made the updates we talked about yesterday and that you requested in this PR. The test passes locally and in semaphore. I validated that the workflow works for CCloud as well.

**Review by bbejeck — APPROVED** [review]
Thanks for the updates @awalther28 LGTM

**Pull request merged**